src/hotspot/cpu/x86/stubGenerator_x86_64.cpp

```diff
@@ -1999,6 +1999,91 @@ address StubGenerator::generate_base64_encodeBlock()
   return start;
 }
 
+address StubGenerator::generate_poly1305_masksCP() {
+  __ align64();
+  StubCodeMark mark(this, "StubRoutines", "generate_poly1305_masksCP");
+  address start = __ pc();
+  // OFFSET 0: high_bit
+  __ emit_data64(0x0000010000000000, relocInfo::none);
+  __ emit_data64(0x0000010000000000, relocInfo::none);
+  __ emit_data64(0x0000010000000000, relocInfo::none);
+  __ emit_data64(0x0000010000000000, relocInfo::none);
+  __ emit_data64(0x0000010000000000, relocInfo::none);
+  __ emit_data64(0x0000010000000000, relocInfo::none);
+  __ emit_data64(0x0000010000000000, relocInfo::none);
+  __ emit_data64(0x0000010000000000, relocInfo::none);
+
+  // OFFSET 64: mask_44
+  __ emit_data64(0xfffffffffff, relocInfo::none);
+  __ emit_data64(0xfffffffffff, relocInfo::none);
+  __ emit_data64(0xfffffffffff, relocInfo::none);
+  __ emit_data64(0xfffffffffff, relocInfo::none);
+  __ emit_data64(0xfffffffffff, relocInfo::none);
+  __ emit_data64(0xfffffffffff, relocInfo::none);
+  __ emit_data64(0xfffffffffff, relocInfo::none);
+  __ emit_data64(0xfffffffffff, relocInfo::none);
+
+  // OFFSET 128: mask_42
+  __ emit_data64(0x3ffffffffff, relocInfo::none);
+  __ emit_data64(0x3ffffffffff, relocInfo::none);
+  __ emit_data64(0x3ffffffffff, relocInfo::none);
+  __ emit_data64(0x3ffffffffff, relocInfo::none);
+  __ emit_data64(0x3ffffffffff, relocInfo::none);
+  __ emit_data64(0x3ffffffffff, relocInfo::none);
+  __ emit_data64(0x3ffffffffff, relocInfo::none);
+  __ emit_data64(0x3ffffffffff, relocInfo::none);
+
+  return start;
+}
+
+address StubGenerator::generate_poly1305_processBlocks() {
+  __ align64();
```

Comment:
This can be replaced by  __ align(CodeEntryAlignment);

Reply:
done